### DISCUSSION TITLE
To much changes so stopped using this

### DISCUSSION THREAD

**Discussion by chrisboer** [djansen1987/SAJeSolar#77] [Ideas]
As SAJ and also other brands like Solarman are continues making changes on their backend servers I just stopped using their backend. 

I have bought 3 smart Shelly relais which perfectly show each day the correct data as SAJ also not showed the correct data sometimes. 
So now I have 4 advantages : 1. I keep data locally    2. no more time wasted in changing infra/config    3. data which I can trust   4. The biggest advantage is to be able to shut down the solar inverters remotely when I have to pay for delivering energy back to the grid. 

So thanks for doing this app but I am off.

**Comment by djansen1987** [comment]
Hi Chris,

That sounds great! Could you provide some more details and maybe photos on how to set it up. Would love to go fully local instead of their cloud.

**Comment by chrisboer** [comment]
![shellypro1pm](https://github.com/djansen1987/SAJeSolar/assets/23163409/46b83db8-0aee-469c-810a-85a5bcd7d02b)
![shelly1pm](https://github.com/djansen1987/SAJeSolar/assets/23163409/e526c69c-e823-4291-a2fe-471a26f81e42)
![solarhut](https://github.com/djansen1987/SAJeSolar/assets/23163409/467c8686-34f8-4097-9a83-f286665f7f08)
![homeyflow](https://github.com/djansen1987/SAJeSolar/assets/23163409/2f17cfe5-6630-4073-9865-4dce4eb3aac4)

Devices can be included in Homey Pro and I made an automated shutdown on negative energy prices to shut down the Shelly Relais fully automated. 

But you can also do this via the Shelly app from anywhere in the world when you don't have a homey.

![shutdown](https://github.com/djansen1987/SAJeSolar/assets/23163409/72d4aead-0c5b-456c-b618-c5b2f9fc0893)
